docs/hugo/content/introduction/authentication.md

```diff
@@ -247,3 +247,65 @@ helm upgrade --install --devel aso2 aso2/azure-service-operator \
      --set aadPodIdentity.azureManagedIdentityResourceId=${IDENTITY_RESOURCE_ID} \
      --set azureClientID=${IDENTITY_CLIENT_ID}
 ```
+
+[test](#using-a-credential-for-aso-with-reduced-permissions)
```

Comment:
Should this have a capital? 

Or maybe it should be removed, since it looks out of place.

Reply:
oops, this was not supposed to be left here. Removed.